src/entities/Ball/Ball.ts

```diff
@@ -1,17 +1,23 @@
 import Phaser from 'phaser';
+import {
+  ballBounce,
+  ballFriction,
+  ballRadius,
+} from '@src/entities/Ball/sliderInputs';
```

Comment:
when did we add this "shorthand" to the config? And I don't see the benefit of it if `@src` is just longer than simply `src`. What about simply `@`?

(Not necessarily relevant for this issue)

Reply:
I sneaked it in when I felt like importing `../../../someFile` felt bad

adjusted the vite and tsconfig accordingly when creating the map:
https://github.com/sebakocz/marblaze/pull/3/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0

I have no real arguments to backup this naming convention, I believe it was just on my whim

just checked and vuejs and pinia use their project names as the main alias

if you wonna change that let me know, I thought @src is good enough

Reply:
Lets keep it for now, no priority